my.css

```diff
@@ -44,55 +42,53 @@ body {
 }
 
 .header__promo {
-  width: 380px;
+  width: 350px;
   display: flex;
   justify-content: space-between;
   align-items: center;
   font-size: 25px;
-
+  padding-top: 40px;
+  padding-left: 30px;
 }
 
 .logo__promo {
-  color: #fff;
-  padding-top: 40px;
-  padding-left: 40px;
+  color: #fff; 
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/itog/pullRequest?prid=5608084)